### PR TITLE
Added check for Archive Of Our Own account.

### PR DESCRIPTION
**Pull request by AccentuSoft**
Hello,

We added a check for the site Archive Of Our Own.

Please let us know if any changes are needed.

### CODE REVIEW

File: web_accounts_list.json [hunk 1]
```diff
@@ -233,6 +234,17 @@
...
      },
+     {
+      "name" : "Archive Of Our Own Account",
+      "check_uri" : "https://archiveofourown.org/users/{account}",
+      "account_existence_code" : "200",
+      "account_existence_string" : ">Profile<",
+      "account_missing_string" : ">redirected<",
+      "account_missing_code" : "302",
+      "known_accounts" : ["test"],
+      "category" : "misc",
```
Comment (WebBreacher) [inline web_accounts_list.json:245]: please change the category to "hobby"

File: web_accounts_list.json [hunk 2]
```diff
@@ -233,6 +234,17 @@
...
       "valid" : true
      },
+     {
+      "name" : "Archive Of Our Own Account",
+      "check_uri" : "https://archiveofourown.org/users/{account}",
+      "account_existence_code" : "200",
+      "account_existence_string" : ">Profile<",
+      "account_missing_string" : ">redirected<",
+      "account_missing_code" : "302",
+      "known_accounts" : ["test"],
```
Comment (WebBreacher) [inline web_accounts_list.json:244]: Please add another test account like `john`

File: wmn-data.json [hunk 3]
```diff
@@ -254,6 +254,18 @@
...
        },
+       {
+        "name" : "Archive Of Our Own Account",
+        "uri_check" : "https://archiveofourown.org/users/{account}",
+        "post_body" : "",
+        "e_code" : "200",
+        "e_string" : ">Profile<",
+        "m_string" : ">redirected<",
+        "m_code" : "302",
+        "known" : ["test"],
```
Comment (WebBreacher) [inline wmn-data.json:265]: same as above.

File: wmn-data.json [hunk 4]
```diff
@@ -254,6 +254,18 @@
...
+        "cat" : "misc",
```
Comment (WebBreacher) [inline wmn-data.json:266]: same as above.

**Review by WebBreacher — CHANGES_REQUESTED** [review]

**Comment by AccentuSoft** [comment]
Done with above ^

**Comment by WebBreacher** [comment]
Thanks for contributing!

**Pull request merged**